### PR TITLE
DOC-334 Add new shard metrics

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

**Comment by bmansheim** [comment]
@oranagra Please review the descriptions in: https://github.com/RedisLabs/redislabs-docs/pull/1013/commits/ec7252b93b522066d13552b3e7e323f86c329bcf

File: content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md [hunk 1]
```diff
@@ -223,32 +223,40 @@ Here are the metrics available to Prometheus:
 | redis_expire_cycle_cpu_milliseconds | The cumulative amount of time spent on active expiry cycles |
 | redis_expired_keys | Keys expired so far (since restart) |
 | redis_forwarding_state | Shard forwarding state (on or off) |
+| redis_keys_trimmed | The number of keys that were trimmed in the current or last resharding process |
 | redis_keyspace_read_hits | Rate of read operations accessing an existing keyspace (ops/sec) |
 | redis_keyspace_read_misses | Rate of read operations accessing an non-existing keyspace (ops/sec) |
 | redis_keyspace_write_hits | Rate of write operations accessing an existing keyspace (ops/sec) |
 | redis_keyspace_write_misses | Rate of write operations accessing an non-existing keyspace (ops/sec) |
```
Comment (oranagra) [inline content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md:230]: i now notice that the hits/misses are mentioning rate (ops/sec).
the metric is a "Running counter" like total_connections_received, redis_rdb_saves, redis_aof_rewrites, and aof_delayed_fsync (which i don't see here, please add)

Comment (oranagra) [reply content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md:230]: the `redis_aof_delayed_fsync` is a counter of how many times an AOF fsync caused delays in the redis main thread (inducing latency). it's an indication that the disk isn't fast enough or is overloaded.

Comment (bmansheim) [reply content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md:230]: @oranagra And what is the aof_delayed_fsync?

File: content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md [hunk 2]
```diff
@@ -224,10 +224,10 @@ Here are the metrics available to Prometheus:
 | redis_expired_keys | Keys expired so far (since restart) |
 | redis_forwarding_state | Shard forwarding state (on or off) |
 | redis_keys_trimmed | The number of keys that were trimmed in the current or last resharding process |
-| redis_keyspace_read_hits | Rate of read operations accessing an existing keyspace (ops/sec) |
-| redis_keyspace_read_misses | Rate of read operations accessing an non-existing keyspace (ops/sec) |
-| redis_keyspace_write_hits | Rate of write operations accessing an existing keyspace (ops/sec) |
-| redis_keyspace_write_misses | Rate of write operations accessing an non-existing keyspace (ops/sec) |
+| redis_keyspace_read_hits | Number of read operations accessing an existing keyspace |
```
Comment (oranagra) [inline content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md:227]: I don't care much but maybe you do.
In one place you use 'Number of' and in another `Running counter of the number of`
maybe you wanna make it consistent and use the shorter form for both..

**Review by oranagra — APPROVED** [review]

**Pull request merged**